modules/nixos/installation/base.nix

```diff
@@ -1,15 +1,36 @@
 { pkgs, config, lib, inputs, system, ... }:
 {
-  boot.kernelPackages = config.boot.zfs.package.latestCompatibleLinuxPackages;
-  environment.sessionVariables.NIXPKGS_ALLOW_UNFREE = "1";
+  # Whitelist wheel users to do anything
+  # This is useful for things like pkexec
+  #
+  # WARNING: this is dangerous for systems
+  # outside the installation-cd and shouldn't
+  # be used anywhere else.
+  security.polkit.extraConfig = ''
+    polkit.addRule(function(action, subject) {
+      if (subject.isInGroup("wheel")) {
+        return polkit.Result.YES;
+      }
+    });
+  '';
+
+  boot.kernelPackages = lib.mkForce config.boot.zfs.package.latestCompatibleLinuxPackages;
```

Comment:
I _think_ this is now handled automatically by the zfs module in NixPkgs.

Reply:
I think you're right, I'll test it out